.github/pull_request_template.md

```diff
@@ -5,3 +5,4 @@ Closes #
 - [ ] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
 - [ ] Rebased/mergeable
 - [ ] Tests pass
+- [ ] E2E Tests pass in k8s-idpe (applicable only if you edited Cypress tests)
```

Comment:
Is there a link to some docs on how to do that?

Reply:
I'll add it to the template, but yes, this: https://docs.influxdata.io/development/guides/local-development/#three-ways-to-run-the-ui-e2e-tests-against-the-kind-cluster